src/ontology/mondo-edit.obo

```diff
@@ -419916,6 +419978,7 @@ name: parturient paresis
 def: "A disease of pregnant and lactating cows and ewes leading to generalized paresis and death. The disease, which is characterized by hypocalcemia, occurs at or shortly after parturition in cows and within weeks before or after parturition in ewes." [MESH:D010319]
 synonym: "animal milk fever" RELATED [MESH:D010319]
 synonym: "fever, animal milk" RELATED [MESH:D010319]
+synonym: "milk fever" EXACT [ISBN-10. 0323088392, ISBN-13. 978-0323088398]
```

Comment:
Bandly formatted ISBN will have caused the error. `ISBN-10:0323088392` if ISBN-10 is a permitted prefix

Reply:
updated